client/vmmCtrlrP_service.go

```diff
@@ -0,0 +1,259 @@
+	}
+	fmt.Printf("%+v", cont)
+
+	return nil
+}
+
+func (sm *ServiceManager) DeleteRelationvmmRsAcc(parentDn string) error {
+	dn := fmt.Sprintf("%s/rsacc", parentDn)
+	return sm.DeleteByDn(dn, "vmmRsAcc")
+}
+func (sm *ServiceManager) CreateRelationvmmRsVmmCtrlrP(parentDn, tDn string) error {
+	dn := fmt.Sprintf("%s/rsvmmCtrlrP-[%s]", parentDn, tDn)
+	containerJSON := []byte(fmt.Sprintf(`{
+		"%s": {
+			"attributes": {
+				"dn": "%s"
+			}
+		}
+	}`, "vmmRsVmmCtrlrP", dn))
+
+	jsonPayload, err := container.ParseJSON(containerJSON)
+	if err != nil {
+		return err
+	}
+
+	req, err := sm.client.MakeRestRequest("POST", fmt.Sprintf("%s.json", sm.MOURL), jsonPayload, true)
+	if err != nil {
+		return err
+	}
+
+	cont, _, err := sm.client.Do(req)
+	if err != nil {
+		return err
+	}
+	fmt.Printf("%+v", cont)
+
+	return nil
+}
+
+func (sm *ServiceManager) DeleteRelationvmmRsVmmCtrlrP(parentDn, tDn string) error {
+	dn := fmt.Sprintf("%s/rsvmmCtrlrP-[%s]", parentDn, tDn)
+	return sm.DeleteByDn(dn, "vmmRsVmmCtrlrP")
+}
+func (sm *ServiceManager) CreateRelationvmmRsVxlanNs(parentDn, tnFvnsVxlanInstPName string) error {
+	dn := fmt.Sprintf("%s/rsvxlanNs", parentDn)
+	containerJSON := []byte(fmt.Sprintf(`{
+		"%s": {
+			"attributes": {
+				"dn": "%s","tnFvnsVxlanInstPName": "%s"
+
+			}
+		}
+	}`, "vmmRsVxlanNs", dn, tnFvnsVxlanInstPName))
+
+	jsonPayload, err := container.ParseJSON(containerJSON)
+	if err != nil {
+		return err
+	}
+
+	req, err := sm.client.MakeRestRequest("POST", fmt.Sprintf("%s.json", sm.MOURL), jsonPayload, true)
+	if err != nil {
+		return err
+	}
+
+	cont, _, err := sm.client.Do(req)
+	if err != nil {
+		return err
+	}
+	fmt.Printf("%+v", cont)
+
+	return nil
+}
+
+func (sm *ServiceManager) DeleteRelationvmmRsVxlanNs(parentDn string) error {
+	dn := fmt.Sprintf("%s/rsvxlanNs", parentDn)
+	return sm.DeleteByDn(dn, "vmmRsVxlanNs")
+}
+func (sm *ServiceManager) CreateRelationvmmRsCtrlrPMonPol(parentDn, tnMonInfraPolName string) error {
+	dn := fmt.Sprintf("%s/rsctrlrPMonPol", parentDn)
+	containerJSON := []byte(fmt.Sprintf(`{
+		"%s": {
+			"attributes": {
+				"dn": "%s","tnMonInfraPolName": "%s"
+
+			}
+		}
+	}`, "vmmRsCtrlrPMonPol", dn, tnMonInfraPolName))
+
+	jsonPayload, err := container.ParseJSON(containerJSON)
+	if err != nil {
+		return err
+	}
+
+	req, err := sm.client.MakeRestRequest("POST", fmt.Sprintf("%s.json", sm.MOURL), jsonPayload, true)
+	if err != nil {
+		return err
+	}
+
+	cont, _, err := sm.client.Do(req)
+	if err != nil {
+		return err
+	}
+	fmt.Printf("%+v", cont)
+
+	return nil
+}
+
+func (sm *ServiceManager) DeleteRelationvmmRsCtrlrPMonPol(parentDn string) error {
+	dn := fmt.Sprintf("%s/rsctrlrPMonPol", parentDn)
+	return sm.DeleteByDn(dn, "vmmRsCtrlrPMonPol")
+}
+func (sm *ServiceManager) CreateRelationvmmRsMgmtEPg(parentDn, tnFvEPgName string) error {
+	dn := fmt.Sprintf("%s/rsmgmtEPg", parentDn)
+	containerJSON := []byte(fmt.Sprintf(`{
+		"%s": {
+			"attributes": {
+				"dn": "%s","tnFvEPgName": "%s"
+
```

Comment:
We usually add "annotation" parameter in the attributes payload with value as "orchestrator:terraform" if relation class support annotation.